cmd/docker-app/main.go

```diff
@@ -2,10 +2,19 @@ package main
 
 import (
 	"os"
+
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/docker/pkg/term"
+	"github.com/sirupsen/logrus"
 )
 
 func main() {
-	cmd := newRootCmd()
+	// Set terminal emulation based on platform as required.
+	stdin, stdout, stderr := term.StdStreams()
+	logrus.SetOutput(stderr)
+
+	dockerCli := command.NewDockerCli(stdin, stdout, stderr, false)
```

Comment:
No content trust for the moment?

Reply:
@silvin-lubecki yes, it wasn't supported on `docker/app` before this PR, it won't after :stuck_out_tongue:

Reply:
Right, but do we want it? Do we need to add an issue just to remember it?

Reply:
@silvin-lubecki in short-term, in long-term, invoking @garethr but I would guess yes :angel:

Reply:
Yes, we'll definitely look at the use of content trust for applications in the future.